plugins/woocommerce-beta-tester/src/live-branches/components/BranchList.tsx

```diff
@@ -2,50 +2,59 @@
  * External dependencies
  */
 import {
-	// @ts-ignore
-	__experimentalItemGroup as ItemGroup,
-	// @ts-ignore
-	__experimentalItem as Item,
 	Button,
+	ComboboxControl,
 } from '@wordpress/components';
+import { useState } from 'react';
 
 /**
  * Internal dependencies
  */
 import { Branch } from '../hooks/live-branches';
 
-const BranchListItem = ( { branch }: { branch: Branch } ) => {
+const BranchInfo = ( { branch }: { branch: Branch } ) => {
 	return (
-		<Item>
+		<div>
```

Comment:
Nitpick but you can avoid rendering anything if you need a wrapper by wrapping your component with `<> </>`